gtsam/linear/NoiseModel.h

```diff
@@ -696,6 +704,14 @@ namespace gtsam {
       virtual void WhitenSystem(Matrix& A1, Matrix& A2, Vector& b) const;
       virtual void WhitenSystem(Matrix& A1, Matrix& A2, Matrix& A3, Vector& b) const;
 
+      virtual Vector unweightedWhiten(const Vector& v) const {
+        return noise_->unweightedWhiten(v);
+      }
+      virtual double weight(const Vector& v) const {
+        // Todo(mikebosse): make the robust weight function input a vector.
```

Comment:
Not really a review point, but can you please explain this TODO and its reasoning to me? Asking so I can take care of this going forward. :slightly_smiling_face: